lib/run.js

```diff
@@ -311,8 +327,9 @@ module.exports = function runBrowser(config, name, version) {
         }
 
         // pass proxy configuration to the new environment
-        if (options.noProxy && env.no_proxy === undefined) {
-          env.no_proxy = options.noProxy;
+        var noProxy = formatNoProxy(options, 'env');
+        if (noProxy && env.no_proxy === undefined) {
+          env.no_proxy = noProxy;
```

Comment:
I don't understand why `no_proxy` is being exposed as an environment variable for the new browser process :thinking:

Reply:
HTTP_PROXY, HTTPS_PROXY and NO_PROXY are standard env vars to set the proxy settings in a given console.

I believe Chrome and Firefox do not honor those, but I don't know if any other browser do.

I left it just to be safe. Shouldn't affect anything.